ecs-cli/modules/commands/flags/flags.go

```diff
@@ -68,6 +68,8 @@ const (
 	ServiceDiscoveryContainerNameFlag           = "sd-container-name"
 	ServiceDiscoveryContainerPortFlag           = "sd-container-port"
 	HealthcheckCustomConfigFailureThresholdFlag = "healthcheck-custom-config-failure-threshold"
+	DeletePrivateNamespaceFlag                  = "delete-namespace"
```

Comment:
it's not apparent from the flag name that it only applies to private namespaces. Do we msg anywhere that deletion of public namespaces is not supported? Is "delete-private-namespace" too verbose?

Reply:
The flag help string is "[Optional] Service Discovery - Deletes the private namespace created by the ECS CLI" 

Do you consider that to be sufficient in terms of alleviating customer confusion?

Reply:
works for me!